github/release_notes/model.py

```diff
@@ -124,7 +140,7 @@ def to_block_str(self):
             '```'.format(
                 cat=self.category_id,
                 t_grp=self.target_group_id,
-                src_repo=self.cn_source_repo.name(),
+                src_repo=self.source_component_name,
```

Comment:
if `src_repo` actually refers to a github-repository, please do not use the component-name / a variable named `component_name`. it is just a coincidence that most (ok, all..) of our component-names are in fact github-repository-URLs (w/o schema). preferrably pass the repository-access. or github-cfg. or the component. not just a str. this is the weakest kind of typing available, and I think we should leverage the benefits of the type-system

I know that the existing codebase is not ideal. however, I would like to at least improve it a bit if we have to touch it anyhow